services/otel-operator/deploy/app.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: otel-operator
+  namespace: monitoring
+spec:
+  interval: 5m0s
+  chart:
+    spec:
+      chart: opentelemetry-operator
+      sourceRef:
+        kind: HelmRepository
+        name: otel-oci
+        namespace: monitoring
+  values:
+    manager:
+      image:
+        repository: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Should we use our internal cache container registry to avoid depending on ghcr.io directly?
```suggestion
        repository: altinncr.azurecr.io/ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

---

is there any rfc or docs on how do we push to that registry?